src/python/era_5g_object_detection_common/requirement.txt

```diff
@@ -1,3 +1,7 @@
 numpy==1.23.*
 opencv_python==4.6.*
-pycocotools==2.0.*
\ No newline at end of file
+pycocotools==2.0.*
+setuptools
+simple-websocket
+gevent-websocket
```

Comment:
gevent-websocket ani eventlet by neměly být potřeba...

Reply:
ty setuptools tady podle mě taky moc nedávají smysl..

Reply:
bez setuptools mi to nejede

Reply:
bez simple-websocket to sice jede, ale vypisuje: 2023-02-16 23:04:21,562 - ERROR - The WebSocket transport is not available, you must install a WebSocket server that is compatible with your async mode to enable it. See the documentation for details. (further occurrences of this error will be logged with level INFO), takže bych to tam nechal

Reply:
simple-websocket určitě nechat